v3/src/components/axis/components/axis-or-legend-attribute-menu.tsx

```diff
@@ -53,14 +53,16 @@ const _AxisOrLegendAttributeMenu = ({ place, target, portal,
   useOutsidePointerDown({ref: menuRef, handler: () => onCloseRef.current?.()})
 
   return (
-    <div className={`axis-legend-attribute-menu ${place}`} ref={menuRef}>
+    <div className={`axis-legend-attribute-menu ${place}`} 
+    data-testid={`axis-legend-attribute-menu-${place}`} ref={menuRef}>
```

Comment:
This probably doesn't matter much, but it doesn't look like this new test ID is used in any tests.

Reply:
Thank you for pointing this out. I've removed this now.